docs/security-policy-for-authors.md

```diff
@@ -0,0 +1,337 @@
+# Security Policy
+
+The purpose of including a Security Policy in a CPAN distribution is
+to advise users how to report security issues, and how these issues
+will be handled.
+
+Status: ⚠️ DRAFT
+
+## The format of the security policy
+
+There security policy should be a text document with minimal
+formatting, either as a plain text `SECURITY` file, a markdown
+`SECURITY.md` file, or a POD format `SECURITY.pod` file.
+
+(Using `SECURITY.txt` or `security.txt` is discouraged, as that is
+often associated with https://securitytxt.org/ which has a different
+purpose.)
+
+The project's `README` file and module documentation should have
+sections that refers users to the security policy document, especially
+in the bug reporting sections.
+
+The file should have the following sections.
+
+1. Abstract
+
+   A short one or two sentences that explain what this document is.
+
+2. How to report a security vulnerability
+
+   How to contact the project maintainers, and what to expect as a reponse.
+
+3. What this policy applies to
+
+   What versions of the software are supported
+
+4. Installation and usage issues
+
+5. Workflow.
+
+## Abstract
+
+This is short one or two sentences that explain what this document is,
+e.g.
+
+> This is the Security Policy for the Perl 5 Foo-Bar distribution.
+
+If the project has a web site, with this security policy posted on the
+website, then you should refer to that URL, e.g.
+
+> The latest version of this Security Policy can be found on the
+> Foo-Bar website at https://foobar.example.com/security-policy
+
+(It is reasonable to link to the project's distribution page on
+MetaCPAN, e.g. https://metacpan.org/dist/Foo-Bar.)
+
+## How to report a security vulnerability
+
+At the bare minimum, the should provide a current package maintainer's
+email address (or ideally an alias that forwards to multiple
+maintainers, or a bug reporting system that supports special handling
+of security issues.
+
+It is important to refer to the current project maintainers, and not
+the package authors, who for older packages may be different.
```

Comment:
This is an interesting point.  If the Spreadsheet::ParseExcel vulnerability had not been copied to the FIRSTCOME, who had not released a version in 11 or 12 years it may have taken longer to get that issue resolved.  The FIRSTCOME responded that they no longer maintained the software which presented an opportunity for "but you could do a release" or assign someone else COMAINT.  They made a release.